mavis/annotate/variant.py

```diff
@@ -692,13 +692,21 @@ def _gather_annotations(ref, bp, proximity=None):
 
         b1_itvl = bp.break1 & a1
         b2_itvl = bp.break2 & a2
+
         bpp = BreakpointPair.copy(bp)
         bpp.break1.start = b1_itvl[0]
         bpp.break1.end = b1_itvl[1]
         bpp.break2.start = b2_itvl[0]
         bpp.break2.end = b2_itvl[1]
 
-        a = Annotation(bpp, a1, a2, proximity=proximity)
```

Comment:
it might be more robust if we put this in the Annotation.__init__.  https://github.com/bcgsc/mavis/blob/7bd6462d431085b0256d3de9333d2d50165bc3bb/mavis/annotate/variant.py#L35-L60

So the breakpoints get swapped when the breakpoint pair is built. We should be able to tell if they've swapped by comparing b1/b2 to self.break1/self.break2

then we would just need to also swap the transcripts